packages/account/src/Types/context.type.ts

```diff
@@ -3,7 +3,4 @@ import { TToken } from './common-prop.type';
 export type TApiContext = {
     api_tokens: NonNullable<TToken[]> | undefined;
```

Comment:
As `api_tokens` is an array of objects, can we please verify if can be undefined or not?
By right, it should be an `empty array` if there's no token instead of `undefined`

It adds an overhead to put null checks while dealing with such type differences

Reply:
Yes, I agree too